--- origsrc/popt-popt-1.18-release/Doxyfile.in	2020-06-23 20:15:00.000000000 +0900
+++ src/popt-popt-1.18-release/Doxyfile.in	2020-12-30 20:52:10.000000000 +0900
@@ -460,14 +460,14 @@ WARN_LOGFILE           =
 # with spaces.
 
 INPUT                  = \
-                        ./popt.c \
-                        ./popt.h \
-                        ./poptconfig.c \
-                        ./popthelp.c \
-                        ./poptint.c \
-                        ./poptint.h \
-                        ./poptparse.c \
-			./system.h
+                        @srcdir@/popt.c \
+                        @srcdir@/popt.h \
+                        @srcdir@/poptconfig.c \
+                        @srcdir@/popthelp.c \
+                        @srcdir@/poptint.c \
+                        @srcdir@/poptint.h \
+                        @srcdir@/poptparse.c \
+			@srcdir@/system.h
 
 # If the value of the INPUT tag contains directories, you can use the 
 # FILE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
