--- origsrc/popt-1.14/Doxyfile.in	2008-02-18 13:11:56.000000000 -0600
+++ src/popt-1.14/Doxyfile.in	2008-04-07 21:41:28.843750000 -0500
@@ -460,15 +460,15 @@
 # with spaces.
 
 INPUT                  = \
-                        ./findme.c \
-                        ./findme.h \
-                        ./popt.c \
-                        ./popt.h \
-                        ./poptconfig.c \
-                        ./popthelp.c \
-                        ./poptint.h \
-                        ./poptparse.c \
-			./system.h
+                        @srcdir@/findme.c \
+                        @srcdir@/findme.h \
+                        @srcdir@/popt.c \
+                        @srcdir@/popt.h \
+                        @srcdir@/poptconfig.c \
+                        @srcdir@/popthelp.c \
+                        @srcdir@/poptint.h \
+                        @srcdir@/poptparse.c \
+                        @srcdir@/system.h
 
 # If the value of the INPUT tag contains directories, you can use the 
 # FILE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
